src/test/scala-2.12/org/apache/james/gatling/simulation/Injection.scala

```diff
@@ -8,6 +8,7 @@ import scala.concurrent.duration._
 object Injection {
 
   private def toUserPerSec(userPerHour: Double): Double = userPerHour / 1.hour.toSeconds
+  def toUsersPerHour(userPerHour: Double): Double = userPerHour * 1.hour.toSeconds
```

Comment:
Isn't it

`def toUsersPerHour(userPerSecond: Double): Double = userPerSecond * 1.hour.toSeconds`

?

Reply:
right now it doesn't make sense, your suggestion is quite likely

Reply:
likely to...?